packages/clerk-js/src/ui/components/UserProfile/MfaTOTPScreen.tsx

```diff
@@ -3,37 +3,30 @@ import React from 'react';
 
 import { useWizard, Wizard } from '../../common';
 import { localizationKeys } from '../../customizables';
-import { SuccessPage, withCardStateProvider } from '../../elements';
+import type { FormProps } from '../../elements';
+import { withCardStateProvider } from '../../elements';
 import { AddAuthenticatorApp } from './AddAuthenticatorApp';
-import { MfaBackupCodeList } from './MfaBackupCodeList';
 import { VerifyTOTP } from './VerifyTOTP';
 
-export const MfaTOTPScreen = withCardStateProvider(() => {
+type MfaTOTPFormProps = FormProps;
+export const MfaTOTPScreen = withCardStateProvider((props: MfaTOTPFormProps) => {
+  const { onSuccess, onReset } = props;
   const wizard = useWizard();
   const ref = React.useRef<TOTPResource>();
 
   return (
     <Wizard {...wizard.props}>
       <AddAuthenticatorApp
         title={localizationKeys('userProfile.mfaTOTPPage.title')}
-        onContinue={wizard.nextStep}
+        onSuccess={wizard.nextStep}
+        onReset={onReset}
       />
 
       <VerifyTOTP
-        onVerified={wizard.nextStep}
+        onSuccess={onSuccess}
+        onReset={onReset}
         resourceRef={ref}
       />
-
```

Comment:
We should keep the `<SuccessPage/>` here as we need to show to the user the backup codes.